### ISSUE TITLE
Make APNGDecoder#getLoopCount public method

### ISSUE THREAD

**Issue by phicdy** [penfeizhou/APNG4Android#206]
### New Issue Checklist

- [x] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Hello @penfeizhou , thank you for awesome library. 
In some cases, I'd like to get loop count, but is not opened. Can we make it public method ? If there are not any concerns, I'll create PR.
https://github.com/penfeizhou/APNG4Android/blob/0d3c268869ed391995f360c7264dea86966a34de/apng/src/main/java/com/github/penfeizhou/animation/apng/decode/APNGDecoder.java#L66-L69
